src/panels/config/person/dialog-person-detail.ts

```diff
@@ -29,6 +29,13 @@ class DialogPersonDetail extends LitElement {
   @property() private _params?: PersonDetailDialogParams;
   @property() private _submitting: boolean = false;
 
+  private _deviceTrackersAvailable = memoizeOne(() => {
```

Comment:
pass the values you need as parameters to `memoizeOne`, if one of those is changed it will recalculated, otherwise return the previous value.
```
  private _deviceTrackersAvailable = memoizeOne((hass) => {
    return Object.keys(hass.states).some(
      (entityId) =>
        entityId.substr(0, entityId.indexOf(".")) === "device_tracker"
    );
  });
```